RetailCoder.VBE/Inspections/InspectionsUI.resx

```diff
@@ -555,4 +555,13 @@
   <data name="VariableTypeNotDeclaredInspectionResultFormat" xml:space="preserve">
     <value>{0} '{1}' is implicitly 'Variant'</value>
   </data>
-</root>
+  <data name="MalformedAnnotationInspectionMeta" xml:space="preserve">
+    <value>An annotation comment is malformed.</value>
```

Comment:
Actually, it was able to "parse" it as such, it is just that the annotation won't work right without the parameters.